src/logic/safe/utils/safeVersion.ts

```diff
@@ -39,23 +28,21 @@ export const safeNeedsUpdate = (currentVersion?: string, latestVersion?: string)
 export const getCurrentSafeVersion = (gnosisSafeInstance: GnosisSafe): Promise<string> =>
   gnosisSafeInstance.methods.VERSION().call()
 
-const checkFeatureEnabledByVersion = (featureConfig: FeatureConfigByVersion, version?: string) => {
-  if (!version) {
-    return false
-  }
-  return featureConfig.validVersion ? semverSatisfies(version, featureConfig.validVersion) : true
+const isEnabledByVersion = (feature: FEATURES, version: string): boolean => {
+  if (!(feature in FEATURES_BY_VERSION)) return true
```

Comment:
If the feature is in the chain features, but not in the versioned map, then we assume it's enabled for all Safe versions.